certified-connectors/Alkymi/readme.md

```diff
@@ -0,0 +1,61 @@
+##Alkymi Connector
+
+Alkymi provides an external API which allows you to easily access the data extracted from unstructured documents 
+processed by the Alkymi app. This connector will allow you to easily pipe data from the external API to any of the apps 
+in the Microsoft PowerApps ecosystem.
+
```

Comment:
@tori-alkymi Please find a template for readme.md file [here](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates) and add the required sections.

Reply:
@tori-alkymi Can you please let us know why this comment was resolved?

Reply:
@tori-alkymi You have missed to add author section